### ISSUE TITLE
Consider performance art

### ISSUE THREAD

**Issue by azaroth42** [linked-art/linked.art#272]
Some examples:

* Guggenheim Panza Collection - documentation/preservation initiative for these types of works
* Timelining by Gerard & Kelly (Guggenheim) - discussed in detail at 1:16:05 in https://www.youtube.com/watch?v=GF3DVVlq83k&feature=youtu.be&t=17935
* Multiple examples in above live stream video of UNFIXED: Material Challenges in Contemporary Art symposium, AIC 2018
* Shylight (Rijksmuseum permanent collection) is a sort of mechanical dance performance
* Maria Hassabi (performance artist) presentation at the UNFIXED symposium (1:58:00)
* https://www.nytimes.com/2018/11/08/t-magazine/tino-sehgal-hirshhorn-museum-art.html

Noted that CRM does not have `Audio- or `AudioVisualItem`, to parallel `VisualItem`. Could be challenging, but is needed.

Tagged defer and use-cases as a reminder to collect more examples and expertise, such that we can hit the ground running when we get to this

**Comment by beaudet** [comment]
should discuss replace defer as a label on this issue now?

**Comment by rpiening** [comment]
Is this issue still deferred? The RKD is currently working on adding performance art to her database. This will require some reworking of our internal data structure and we'd like to anticipate the translation to linked (art) data in parallel. 

It's the (concept of the) performance itself our art historians want to describe, not the registration (video/photo/otherwise) of it. We'd prefer to stick as closely as possible to the pattern used for other artworks, but "HumanMadeObject" is perhaps a bit of a stretch. Does Linked Art have a stance on this yet? Would also love to discuss this further within this group. 

As for examples, I haven't seen any performance art modeled in cidoc-crm yet? There is however a [Dutch effort to model 'stage arts' in RDA](https://github.com/PodiumkunstNet/ApplicationProfileRDA/blob/main/rdf/examples/09_zeemaal.md) which is very interesting.

PS the same question goes for conceptual art

**Comment by azaroth42** [comment]
Happy to discuss here in the comments, or on the calls once we have 1.0 finalized (or no issues left to discuss for it).

My opinion on modeling:
* The performance is an Activity, per https://linked.art/api/1.0/endpoint/event/, with a time, place, group/person that carried it out and so on. 
* The activity then has some relationship to an abstract work in the same way as an exhibition. Thus the artist carries out the creation of the abstract work, and the performer carries out the performance of it. There might need to be a new linked art relationship minted here -- performance_of or similar
* It could be recorded (video, photo, etc) which would produce a VisualItem and/or LinguisticObject, that was then carried by the physical (DVD) or digital (mp4 file) object. 
* You would then "accession" the abstract work or the recording of the performance, or assign some copyright or other legal rights to the abstract work if the work was "for hire"

I believe this would scale to other types of performance as well, such as theatre, opera, dance or similar.

In this way you can have different performers carrying out the same performance art piece at different times and places. My favorite one is the when the performer is in the guise of a security guard in a regular gallery but dances and does other non-security-guard like things :)

**Comment by beaudet** [comment]
for compare / contrast / discussion, would an example of a photographic print created from a negative might provide a physical parallel to the conceptual object in the performance realm so the JSON modeling of each can be seen together?

**Comment by beaudet** [comment]
So it sounds correct to say that the main difference between modeling performance art vs. art hanging on walls is the conceptual vs. physical difference and the associated CDOC classes that address these entities differently?

**Comment by azaroth42** [comment]
Yep -- the core philosophical difference is that a painting is completely present at any given point in time, whereas at a mathematically precise point in time performance art can only be said to be in progress or not -- an endurant versus a perdurant. Performance art instances have a beginning time and end time, and each then instantiates some abstract set of stage directions or concept (depending on how convoluted it is, and how much is up to the performer), whereas physical art objects can be created and destroyed, but otherwise are (modulo conservation or vandalism) static things.

Which is why we've deferred discussion of them, as they're quite separate from physical art, and more akin to the (much easier to discuss) exhibitions and other events.

**Comment by rpiening** [comment]
Interesting points. Good to know in advance that per LA, performance art falls into an entirely different pattern than physical artworks. To confirm my understanding, the abstract work of the performance would be a propositional object, as per [the base model](https://linked.art/model/base/)?

**Comment by clarisseb** [comment]
I would be happy to contribute with use cases. My team and I have just done a state of the art of ontologies for performing arts. Long story short we plan to use Linked Art for a research project called Stage, starting with the Festival d'Avignon as the main corpus.
